src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -906,6 +907,25 @@ export default class ElementWrapper extends Wrapper {
 		});
 	}
 
+	add_styles(block: Block) {
+		this.node.styles.forEach(style_directive => {
+			
+			// @FIXME:
+			// I'm not sure how to set the right value expression here
+			// Hmm, I thought this would work:
+			
+			// const { name, expression: { node: expression } } = style_directive;
+			// const updater = b`@set_style(${this.var}, "${name}", ${expression}, false)`;
```

Comment:
i think it should be `style_directive.expression.manipulate(block)` instead of `style_directive.expression.node`